custom/icds_reports/migrations/0071_visits_not_vists.py

```diff
@@ -15,4 +15,4 @@ class Migration(migrations.Migration):
 
     operations = [
     ]
-    operations.extend(get_view_migrations())
+    operations.extend(get_view_migrations()[:-1])
```

Comment:
why does a pre-existing migration need to be modified?

Reply:
This function is running SQL that generates/ refreshes SQL views. This migration needs 4 views to be generated to work properly, yet we can not keep all as we are adding to the ChildHealthMonthlyView a new column which source is in a column that is added in later migration and would cause failure.